daemon/endpoint_test.go

```diff
@@ -62,17 +63,26 @@ func (ds *DaemonSuite) TestEndpointAdd(c *C) {
 }
 
 func (ds *DaemonSuite) TestEndpointAddNoLabels(c *C) {
+	// Create the endpoint without any labels.
 	epTemplate := getEPTemplate(c)
 	code, err := ds.d.createEndpoint(epTemplate, strconv.FormatInt(epTemplate.ID, 10), nil)
 	c.Assert(err, IsNil)
 	c.Assert(code, Equals, apiEndpoint.PutEndpointIDCreatedCode)
 
+	// Check that the endpoint has the reserved:init label.
 	ep, err := endpointmanager.Lookup(strconv.FormatInt(epTemplate.ID, 10))
 	c.Assert(err, IsNil)
 	expectedLabels := labels.Labels{
 		labels.IDNameInit: labels.NewLabel(labels.IDNameInit, "", labels.LabelSourceReserved),
 	}
 	c.Assert(ep.OpLabels.IdentityLabels(), comparator.DeepEquals, expectedLabels)
+
+	// Check that the policy is always enforced for endpoints with the
+	// reserved:init label, regardless of the mode that is configured.
+	policy.SetPolicyEnabled(endpoint.NeverEnforce)
```

Comment:
This shouldn't be the case in my opinion. If policy enforcement for the daemon is set to `NeverEnforce`, then the policy corresponding to `reserved:init` should not apply at all. In other words, the policy applying to `reserved:init` should behave just like any other policy in conduit with the daemon's policy enforcement mode.

Reply:
Fixed.